.travis.yml

```diff
@@ -12,16 +12,16 @@ matrix:
   include:
     - os: linux
       compiler: clang
-      env: BTYPE=RelWithDebInfo COV=OFF DEPLOY=yes
+      env: BTYPE=Release COV=OFF DEPLOY=yes
```

Comment:
Why did you change these from RelWithDebInfo to Release? (I don't have an objection, just curious -- seems unrelated to the rest of the PR.)

Reply:
This change was actually from a PR I created and that is now merged. I'm not sure why it showed up in this PR's diff.

Reply:
Hmmm. There were several other places that looked like spurious changes in this PR, which includes a surprisingly large number of commits. Possibly it needs to be merged with or rebased on master.

Also, the GitHub reviewing tool seems really awful. We should switch to Reviewable!